tests/src/Traits/CckContainerTrait.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\Tests\oe_whitelabel\Traits;
+
+/**
+ * Contains methods to assert the presence of specific blocks and elements.
+ */
+trait CckContainerTrait {
+
+  /**
+   * Asserts the HTML of the "cck_here" container.
+   */
+  protected function assertCckContainer(): void {
+    $cck_container = $this->cssSelect('#cck_here')[0];
```

Comment:
```suggestion
    $cck_container = $this->cssSelect('body :nth-child(2)')[0];
```

Reply:
This way we check the location too, we are moving to second place inside body.
Since we are doing the check in the setUp, toolbar shouldn't be a problem, unless we log in with a user before calling the paren't setUp.

Reply:
done, indeed, thanks